engines/order_management/app/services/order_management/order/updater.rb

```diff
@@ -52,11 +41,47 @@ def update
       # - total - order total, it's the equivalent to item_total plus adjustment_total
       def update_totals
         order.payment_total = payments.completed.sum(:amount)
-        order.item_total = line_items.map(&:amount).sum
+        update_item_total
+        update_adjustment_total
+        update_order_total
+      end
+
+      # Give each of the shipments a chance to update themselves
+      def update_shipments
+        shipments.each { |shipment| shipment.update!(order) }
+      end
+
+      def update_item_total
+        order.item_total = line_items.sum('price * quantity')
```

Comment:
You changed this because of performance, I guess.

Reply:
It's better performance yeah, but this change is also added in Spree 2.2 :+1: